subscription_oca/models/account_move.py

```diff
@@ -10,3 +10,11 @@ class AccountMove(models.Model):
     subscription_id = fields.Many2one(
         comodel_name="sale.subscription", string="Subscription"
     )
+
+    def action_open_subscription(self):
+        self.ensure_one()
+        action = self.env["ir.actions.act_window"]._for_xml_id(
+            "subscription_oca.sale_subscription_action"
+        )
+        action["domain"] = [("id", "=", self.subscription_id.id)]
```

Comment:
I suppose the goal is to display the related form view.
You can use:
`return self.subscription_id.get_formview_action()`